Providers/Modules/Plugins/SecurityBaseline/conf/security_baseline.conf

```diff
@@ -10,7 +10,7 @@
 <source> 
     type exec
     tag oms.security_baseline
-    command /opt/microsoft/omsagent/plugin/omsbaseline -d /etc/opt/microsoft/omsagent/conf/omsagent.d/ 
+    command sleep 60 && /opt/microsoft/omsagent/plugin/omsbaseline -d /etc/opt/microsoft/omsagent/conf/omsagent.d/ 
```

Comment:
Have you tested that this resolves the issue on a machine that repro'd the error that customers were seeing?